docs/extensions/logging-and-monitoring.md

```diff
@@ -90,31 +97,34 @@ data:
+- One Loki statefulset in the `garden` namespace, which contains logs from the seed cluster and one per each shoot namespace, which contains logs from shoot's control plane.
+- One Grafana Deployment in the `garden` namespace and two Deployments per shoot namespace (one exposed to the end users and one for the operators). Grafana is the UI component used to explore monitoring and log data together for easier troubleshooting and in context.
 
-:warning: As there is only the central fluent-bit daemonset and not a shoot-specific deployment like in the case of monitoring, see [above](#monitoring), the logging parse configuration must be only provided once and **not per shoot namespace**.
-Also, as fluent-bit parses the logs based on the container name you should make sure that the container names inside your provider-specific pods are prefixed with your extension name.
+Logs can be produced from various sources, such as containers or systemd, and in different formats. The fluent-bit design supports configurable [data pipeline](https://docs.fluentbit.io/manual/concepts/data-pipeline) to address that problem. Gardener provides such [configuration](https://github.com/gardener/gardener/blob/master/charts/seed-bootstrap/charts/fluentd-es/templates/fluent-bit-configmap.yaml) for logs produced by all its core managed components as a `ConfigMap`. Extensions can contribute their own, specific configurations as `ConfigMap`s too. See for example the [logging configuration](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/configmap-logging.yaml) for the Gardener AWS provider extension. The Gardener reconciliation loop watches such resources and updates the fluent-bit agents dynamically.
 
-### Submitting logging parse configuration?
+### Extensions logging integration
 
-To submit a new logging parse configuration, create a `ConfigMap` labeled `extensions.gardener.cloud/configuration=logging` with a single data field - `filter-kubernetes.conf`, containing the fluent-bit configuration for parsing container logs. Before deploying the central fluent-bit instances into the `garden` namespace, Gardener will read all `ConfigMap`s in `garden` with that label and apply the configuration encoded in them.
-Such `ConfigMap`s may contain a single field in their `data`:
+To integrate with Gardener logging, extensions can and *should* specify how fluent-bit will handle the logs produced by the managed components that they contribute to Gardener. Normally, that would require to configure a *parser* for the specific logging format, if none of the available is applicable, and a *filter* defining how to apply it. For a complete reference for the configuration options, refer to fluent-bit's [documentation](https://docs.fluentbit.io/manual/). Note that at the moment only *parser* and *filter* configurations are supported.
```

Comment:
Separate the last sentence as a Note: on a new line:
**Note:** At the moment only *parser* and *filter* configurations are supported.